outputs.tf

```diff
@@ -17,3 +17,13 @@ output "dl_queue_url" {
   description = "Dead letter queue url."
   value       = one(aws_sqs_queue.dlqueue[*].url)
 }
+
+output "lambda_function_arn" {
+  description = "Lambda function arn."
+  value       = one(aws_lambda_function.this[*].lambda_function_arn)
```

Comment:
As the `aws_lambda_function` resource is not optional, the `one` function is not req'd.  Could we please update this to a std ref to a resource?.  This also applies to the `lambda_function_name` output.

TIA